### PR TITLE
[MO] - [system test] -> Fix race-condition for mirroring topics

### PR DESCRIPTION
**Pull request by see-quick**
Signed-off-by: morsak <xorsak02@stud.fit.vutbr.cz>

### Type of change

- Bugfix

### Description

This PR solving race-condition for mirroring topics and also changing name of the mirrored topics. 

### Checklist

- [ ] Make sure all tests pass

### CODE REVIEW

**Comment by Frawless** [comment]
@strimzi-ci run tests test_only progile=regression testcase=OauthPlainST

File: systemtest/src/test/java/io/strimzi/systemtest/oauth/OauthPlainST.java [hunk 1]
```diff
@@ -193,10 +195,18 @@ void testProducerConsumerMirrorMaker() throws IOException, InterruptedException,
...
-        assertThat(consumer.get(Constants.GLOBAL_CLIENTS_TIMEOUT, TimeUnit.MILLISECONDS), is(MESSAGE_COUNT));
+        TestUtils.waitFor("Waiting for Mirror Maker will copy messages from " + CLUSTER_NAME + " to " + targetKafkaCluster,
+            Duration.ofSeconds(30).toMillis(), Constants.TIMEOUT_FOR_MIRROR_MAKER_COPY_MESSAGES_BETWEEN_BROKERS,
+            () -> {
+                try {
+                    Future<Integer> oauthConsumer = oauthKafkaClient.receiveMessages(TOPIC_NAME, NAMESPACE, targetKafkaCluster,
+                            MESSAGE_COUNT, CONSUMER_GROUP_NAME + "-" + rng.nextInt(Integer.MAX_VALUE));
+                    return MESSAGE_COUNT == oauthConsumer.get(Constants.GLOBAL_CLIENTS_TIMEOUT, TimeUnit.MILLISECONDS);
+                } catch (IOException | InterruptedException | ExecutionException | TimeoutException e) {
+                    LOGGER.info("Consumer doesn't received messages {}", e.getMessage());
```
Comment (ppatierno) [inline systemtest/src/test/java/io/strimzi/systemtest/oauth/OauthPlainST.java]: out of curiosity, why an exception is logged as INFO and not as ERROR printing the stack trace? Is it the pattern that you are using everywhere?

File: systemtest/src/test/java/io/strimzi/systemtest/oauth/OauthPlainST.java [hunk 2]
```diff
@@ -276,8 +287,18 @@ void testProducerConsumerMirrorMaker2() throws IOException, InterruptedException
                 .endSpec()
                 .done();
 
-        consumer = oauthKafkaClient.receiveMessages(TOPIC_NAME, NAMESPACE, kafkaTargetClusterName, MESSAGE_COUNT,
-                CONSUMER_GROUP_NAME + "-" + rng.nextInt(Integer.MAX_VALUE));
+        TestUtils.waitFor("Waiting for Mirror Maker will copy messages from " + kafkaSourceClusterName + " to " + kafkaTargetClusterName,
```
Comment (ppatierno) [inline systemtest/src/test/java/io/strimzi/systemtest/oauth/OauthPlainST.java]: Make it clear it's Mirror Maker 2 in the log message.

File: systemtest/src/test/java/io/strimzi/systemtest/oauth/OauthPlainST.java (hunk 1 above)
Comment (see-quick) [reply systemtest/src/test/java/io/strimzi/systemtest/oauth/OauthPlainST.java]: Good point, It will make more sense if I change it to error :)

**Pull request merged**